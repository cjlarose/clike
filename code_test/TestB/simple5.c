
void printdouble(double);
void printchar(char);

double x;

void main()
{

  double y,z;
  double a;

  x = 3.0;
  y = 4.5;
  z = 5.0;
 
  a = x*y + z;
  printdouble(a);
  printchar(10);
}
  
