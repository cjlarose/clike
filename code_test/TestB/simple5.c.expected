Loaded: /usr/local/share/spim/exceptions.s
18.5
